utils/update_checkout/update-checkout-config.json

```diff
@@ -301,37 +300,11 @@
                 "cmake": "v3.16.5",
                 "indexstore-db": "release/5.3",
                 "sourcekit-lsp": "release/5.3",
-                "swift-format": "master",
+                "swift-format": "swift-5.3-branch",
```

Comment:
One thing you should probably do is mark any changes that you make to Apple files. This makes it *way* easier to apply deltas to a new branch. I block in the changes with a comment like
```
// hawley was here
...changes I've made
// end hawley

Reply:
@stephen-hawley it looks like I cannot make comments in this file since it is a json. Or did you mean in the PR notes?

Reply:
Yeah, thanks to JSON no comments :(

Reply:
GRRR.